app/assets/javascripts/modules/Filter.js

```diff
@@ -0,0 +1,10 @@
+define(['jquery', 'List', 'ListFuzzySearch'], function($, List, ListFuzzySearch) {
```

Comment:
(Extra points available for having this boot from a data attribute using the `componentLoader` and inherit from `DoughBaseComponent`, though. [Example here](https://github.com/moneyadviceservice/pensions_calculator/blob/master/app/assets/javascripts/pensions_calculator/modules/FastClick.js).)